homeassistant/components/imgw_pib/config_flow.py

```diff
@@ -58,7 +60,10 @@ async def async_step_user(
                 return self.async_create_entry(title=title, data=user_input)
 
         try:
-            imgwpib = await ImgwPib.create(client_session)
+            imgwpib = await ImgwPib.create(
+                client_session,
+                hydrological_details=False,
```

Comment:
Adding missing `hydrological_details` parameter, relates to https://github.com/home-assistant/core/pull/134668